results/ZMK/MDBT50Q-RX-nRF52840/README.md

```diff
@@ -0,0 +1,4 @@
+Test setup:
+
+- Host: Raspberry Pi 4 running Raspberry OS version 12
+- Keyboard: MDBT50Q-RX dongle as a 1 key keyboard
```

Comment:
I think this needs more info. In particular,
* What version of ZMK was used? One of the test files refers to "LLPM", and two refer to "ESB" (enhanced shockburst?) so this seems to be a custom version of ZMK? To make it clear that this is not mainline ZMK, I think we should
  * Link to the repo and commit from which the firmware was built.
  * Move this to a new folder such as "ZMK (experimental)" to clarify that it is a custom version of ZMK.
* This seems to indicate you were using dongle mode, but there is no information about what was connected to the dongle here.
* Only one of the test files has "dongle" in the name though. Were the LLPM/ESB tests done using dongle mode?
* What was the setup for the USB tests? Was that just the dongle generating its own key presses, with no connection to a separate keyboard? If so, this seems like it may be redundant with the nRF5340 DK data. (The only differences would be the processor and the number of keys, and the results are close enough to one another that they could just be caused by differences in our test hardware rather than anything meaningful.)
* What was the setup for the BLE test without "dongle" in the name? Was that just the MDBT50Q-RX dongle acting as a full keyboard and connected to the Raspberry Pi via BLE? If so, this *may* be redundant with the nRF5340 DK data, though the data is noticeably different. Did you have the Raspberry Pi inside a case or anything else that might affect the wireless signal? If not, then this data looks useful to keep, since it shows how different hardware designs might perform differently given the same Bluetooth settings.

Reply:
- Will move these to ZMK experimental and link my fork
- Will add details to the readme for each experiment and what was connected to what
- plain BLE test was dongle plugged into the RPi (no case) but sending keypresses over bluetooth

Would you prefer more verbose csv file names to show up in the graph or just expand on each one in the hardware's README?